docs/reference/changelog.rst

```diff
@@ -70,6 +70,20 @@ Changelog
 
   For a complete list of commits, check out the `X.Y.Z`_ release on GitHub.
 
+8.6.2 (2025-Feb-14)
+-------------------
+
+Core
+====
+
+* Fix a bug where LXD instances would fail during setup, due to a race
```

Comment:
I'm not sure if that comma should be there.

Reply:
Agreed.